clio-client/src/test/scala/org/broadinstitute/clio/client/dispatch/MoveGvcfSpec.scala

```diff
@@ -28,31 +28,18 @@ class MoveGvcfSpec extends BaseClientSpec {
     recoverToSucceededIf[Exception] {
       val command =
         MoveGvcf(
-          transferGvcfV1Key = testGvcfTransferV1Key,
+          key = testGvcfTransferV1Key,
           destination = testGvcfCloudSourcePath
         )
       succeedingDispatcher.dispatch(command)
     }
   }
 
-  it should "throw an exception if the source and destination paths are the same" in {
```

Comment:
I didn't see much point in this test, but can re-add it if we want it.

Reply:
It's worth keeping if it's testing code we wrote to check for this case. If the exception is being generated by some underlying behavior of APIs we didn't write then I would delete it.

Reply:
OH I know what this is for, it's to protect us from accidentally deleting things (since we copy then delete the old, if new = old then we'll end up just deleting).

I'll add this back in.

Reply:
Hrm, or should we just detect when source == destination and do nothing?